make_functions.sh

```diff
@@ -155,14 +155,14 @@ build_push_operator_image() {
 seed_repository() {
   set -x
   "$DOCKER_BIN" pull "docker.io/jujusolutions/juju-db:${JUJU_DB_VERSION}"
```

Comment:
This will need to change too - all the docker.io references

Reply:
I thought we might want to leave this as docker for the CI mirror test, I'll refactor this accordingly then, thanks Ian!